client/components/App.jsx

```diff
@@ -1,16 +1,16 @@
 import React from 'react'
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
-
+import { BrowserRouter as Router, Switch } from 'react-router-dom'
+import Dashboard from './Dashboard'
```

Comment:
Please put one line separating module imports and component imports. 
Importing `react` and `react-router-dom` should be one line after each other.